### DISCUSSION TITLE
Open request for GIFs or Screenshots! 📸

### DISCUSSION THREAD

**Discussion by winstxnhdw** [winstxnhdw/lc-hax#153] [Announcements]
If you’ve got a GIF or screenshot you’d like to share, please post them under this thread! 

Your post may end up featured on our README!

cc: @joep26020, @yujispin990, @Quesoteric

**Comment by yujispin990** [comment]
ooooh ill try! i only play at night though 😅

**Reply by winstxnhdw** [reply]
Take your time!

**Reply by yujispin990** [reply]
I did send 2 people to the void, make them fight to the death with a shovel and whoever wins gets teleported back to the ship and gets a 10,000 credit prize XD then i leave. I could've recorded while being a phantom showing the fight.

**Reply by yujispin990** [reply]
Sadly i am not very creative, hopefully the other 2 could provide great content. That said, i will try my best to get something good!

**Reply by yujispin990** [reply]
Also thank you @winstxnhdw @LTYGUY for the lc-hax project ❤️  you guys!

**Comment by Quesoteric** [comment]
![crosshair](https://github.com/winstxnhdw/lc-hax/assets/140112866/97cab8cd-32a4-460f-a64b-ebc7d93e77b1)

**Reply by Quesoteric** [reply]
Also quick request, for the widescreen gifs that aren't shits & giggles like this, could you add something like a `/clear` command to get rid of all the messages in chat?

**Reply by winstxnhdw** [reply]
Easily.

**Reply by winstxnhdw** [reply]
Almost forgot about this, but I've added `/clear`. I can't test it yet so let me know if there's any issues.

**Reply by Quesoteric** [reply]
Rapid fire

![rapidfire](https://github.com/winstxnhdw/lc-hax/assets/140112866/893259c4-8fee-4c8f-8799-bd3dd15c5ca2)

I'll do mostly just the basic features of the mod, I probably won't upload any funny usage gifs like the gladiator and nutcracker ones

**Reply by winstxnhdw** [reply]
sick

**Reply by Quesoteric** [reply]
phantom

![ezgif com-overlay](https://github.com/winstxnhdw/lc-hax/assets/140112866/1ba33cdf-e5b6-416e-9419-eda174bab86f)

**Reply by joep26020** [reply]
that could be used as a title gif !

**Reply by winstxnhdw** [reply]
You are right, I'll add it to the title tomorrow.

**Reply by Quesoteric** [reply]
godmode, even though every person who has played a computer game knows what it means and what it will do

![ezgif-5-83cfb8b496](https://github.com/winstxnhdw/lc-hax/assets/140112866/4ba88453-abf9-4c62-8f62-624bc1b1ffa5)

![6d3](https://github.com/winstxnhdw/lc-hax/assets/140112866/af66a4ae-700d-48dd-801d-a24fc9a3ea31)

**Reply by Quesoteric** [reply]
Infinite interact range

_i can't get over this mod it's too funny lol, i'm sending just the video file here in case you want to remove it or keep it yourself when GIF-ifying it_



https://github.com/winstxnhdw/lc-hax/assets/140112866/4383d7b4-65bf-41bf-9f60-4c03fca08c1d

**Reply by winstxnhdw** [reply]
> godmode, even though every person who has played a computer game knows what it means and what it will do

Nanomachines, son!

**Reply by winstxnhdw** [reply]
> Infinite interact range
> 
> 
> 
> _i can't get over this mod it's too funny lol, i'm sending just the video file here in case you want to remove it or keep it yourself when GIF-ifying it_
> 
> 
> 
> 
> 
> 
> 
> https://github.com/winstxnhdw/lc-hax/assets/140112866/4383d7b4-65bf-41bf-9f60-4c03fca08c1d

That mod is amazing, they even animated the HUD LMAO

**Comment by yujispin990** [comment]
![gladiator](https://github.com/winstxnhdw/lc-hax/assets/20551698/387637b6-658d-4e22-9ff0-49212649c1a5)
Gladiator minigame!

**Reply by yujispin990** [reply]
I promised whoever wins gets teleported back to the ship and gets a winning prize! 😆

**Reply by yujispin990** [reply]
Anti-Kick really helps alot, Thanks @armorfid @winstxnhdw  and @LTYGUY !

**Reply by joep26020** [reply]
where do you find that place lmao

**Comment by joep26020** [comment]
![Lethal Company Screenshot 2024 01 03 - 20 28 57 53](https://github.com/winstxnhdw/lc-hax/assets/143763241/01e12e07-c42b-441b-8285-f5056b2f905e)
the trap

**Reply by winstxnhdw** [reply]
LMAO, I used to do this too.

**Comment by joep26020** [comment]
![ezgif-3-7a947792bc](https://github.com/winstxnhdw/lc-hax/assets/143763241/a2862365-71c9-4d22-9d73-421fa9eab8d0)
the nutcracker's spree
Had to infinite jump him out of the inside. Btw you just m3 him while alive, thanks to https://github.com/winstxnhdw/lc-hax/issues/168#issuecomment-1877896450 for the tips!
P.S. you have to be alive, and his kick doesn't work.

**Reply by joep26020** [reply]
@winstxnhdw

**Reply by winstxnhdw** [reply]
Holy, this is good. Do you have the original video for it? I'll render a high-definition GIF of this.

**Reply by joep26020** [reply]
Yes I do. How would I send it to you?

**Reply by winstxnhdw** [reply]
I believe you can just upload the video directly here. If you have Telegram, you can send it to me directly through there as well.

**Reply by joep26020** [reply]
limit is 10mb here. whats ur telegram?

**Reply by winstxnhdw** [reply]
https://t.me/winsstooon

**Comment by joep26020** [comment]
![funnytur](https://github.com/winstxnhdw/lc-hax/assets/143763241/ecc0b1bc-da30-4ef1-9884-67da2ee50ba8)
Glitched seed so u can see turrets easier on brutal company!
